sematic/ee/plugins/publishing/github/endpoints.py

```diff
@@ -0,0 +1,69 @@
+"""Metadata about the server itself."""
+
+# Standard Library
+from dataclasses import asdict
+from logging import getLogger
+from typing import Optional
+
+# Third-party
+import flask
+
+# Sematic
+from sematic.api.app import sematic_api
+from sematic.api.endpoints.auth import authenticate
+from sematic.db.models.git_info import GitInfo
+from sematic.db.models.user import User
+from sematic.ee.plugins.publishing.github.check import check_commit
+
+logger = getLogger(__name__)
+
+
+@sematic_api.route(
+    "/api/v1/github/commit-check/<owner>/<repo>/<commit_sha>",
```

Comment:
Thoughts on namespacing plug-ins endpoints with `/api/v1/plugins/`:
```
/api/v1/plugins/github/commit-check/<owner>/<repo>/<commit_sha>
```
I have no strong opinion, just throwing it out there. This would relieve plug-ins from having to play nice with other endpoints but 🤷

Reply:
Might make sense, but the existing convention (based off the local storage plugin) seems to be that we don't do that:

https://github.com/sematic-ai/sematic/blob/main/sematic/plugins/storage/local_storage.py#L91

And it seems fine to me to do it this way, since the plugin should include its own name in the url (ex: `/github` here).So I'll leave it as-is until somebody has a strong opinion.